problems/2021-cnrs-benchmark/phase-1/full-coupling.i

```diff
@@ -0,0 +1,314 @@
+density = .002  # kg cm-3
+cp = 3075       # J kg-1 K-1, 6.15 / 2.0e-3
+k = .005        # W cm-1 K-1
+gamma = 1       # W cm-3 K-1, Volumetric heat transfer coefficient
+viscosity = .5  # dynamic viscosity
+alpha = 1       # SUPG stabilization parameter
+t_alpha = 2e-4  # K-1, Thermal expansion coefficient
```

Comment:
where did you get this number from?

Reply:
I am curious also about how the benchmark exercise is defined.
Is it defined to be solved with the Boussinesq approximation or to solve the density equation as well?
How does CNRS solve it?
@smpark7

Reply:
Hey @robfairh! `gamma`, `viscosity`, and `t_alpha` are all directly from the CNRS benchmark journal article linked above. The authors picked a 2m-by-2m 2D domain for the geometry. All co-authors used incompressible Navier-Stokes with the Boussinesq approximation for their own multiphysics simulation tools.